editor/code_editor.cpp

```diff
@@ -916,6 +916,13 @@ void CodeTextEditor::input(const Ref<InputEvent> &event) {
 		accept_event();
 		return;
 	}
+	// Trigger script validation when the "Save" keybinding is pressed.
+	// TODO: Figure out how to make saving via the "File" menu trigger script validation aswell.
```

Comment:
```suggestion
	// TODO: Figure out how to make saving via the "File" menu triggers script validation as well.
```

Reply:
I amended the comment; I didn't know about the pre-commit checks until after I made the PR. Woops!

That said, I don't think "triggers" is right?